src/AssemblyGenerator.Constructors.cs

```diff
@@ -1,129 +1,50 @@
-﻿using System;
-using System.Linq;
+﻿using System.Collections.Generic;
 using System.Reflection;
-using System.Reflection.Metadata;
 using Lokad.ILPack.IL;
 
 namespace Lokad.ILPack
 {
     public partial class AssemblyGenerator
     {
-        internal BlobHandle GetConstructorSignature(ConstructorInfo constructorInfo)
+        private void CreateConstructor(ConstructorInfo ctor)
         {
-            var parameters = constructorInfo.GetParameters();
-            var countParameters = parameters.Length;
-
-            var blob = BuildSignature(x => x.MethodSignature(
-                    ConvertCallingConvention(constructorInfo.CallingConvention),
-                    isInstanceMethod: !constructorInfo.IsStatic)
-                .Parameters(
-                    countParameters,
-                    r => r.Void(),
-                    p =>
-                    {
-                        foreach (var par in parameters)
-                        {
-                            var parEncoder = p.AddParameter();
-                            parEncoder.Type().FromSystemType(par.ParameterType, this);
-                        }
-                    }));
-            return GetBlob(blob);
-        }
-
-        internal EntityHandle? GetTypeConstructor(Type type)
-        {
-            return CreateConstructorForReferencedType(type);
-
-            //_ctorHandles.TryGetValue(type, out var collection);
```

Comment:
Idem

Reply:
It should be removed since it has no purpose now.